plugin/signify.vim

```diff
@@ -38,13 +38,14 @@ augroup END
 
 " Init: commands {{{1
 
-command! -nargs=0 -bar SignifyDebug           call sy#debug#list_active_buffers()
-command! -nargs=0 -bar SignifyDebugDiff       call sy#debug#verbose_diff_cmd()
-command! -nargs=0 -bar SignifyDebugUnknown    call sy#repo#debug_detection()
-command! -nargs=0 -bar SignifyFold            call sy#fold#do()
-command! -nargs=0 -bar SignifyRefresh         call sy#util#refresh_windows()
-command! -nargs=0 -bar SignifyToggle          call sy#toggle()
-command! -nargs=0 -bar SignifyToggleHighlight call sy#highlight#line_toggle()
+command! -nargs=0 -bar       SignifyDebug           call sy#debug#list_active_buffers()
+command! -nargs=0 -bar       SignifyDebugDiff       call sy#debug#verbose_diff_cmd()
+command! -nargs=0 -bar       SignifyDebugUnknown    call sy#repo#debug_detection()
+command! -nargs=0 -bar       SignifyFold            call sy#fold#do()
+command! -nargs=0 -bar       SignifyRefresh         call sy#util#refresh_windows()
+command! -nargs=0 -bar       SignifyToggle          call sy#toggle()
+command! -nargs=0 -bar       SignifyToggleHighlight call sy#highlight#line_toggle()
+command! -nargs=0 -bar -bang SignifyToggleFold      call sy#fold#toggle(<q-bang>)
```

Comment:
A more typical way of doing this is `call sy#fold#toggle(<bang>0)`.  Then the argument to `sy#fold#toggle` can be named `newtab`, which will have a value of 0 when `SignifyToggleFold` is called and 1 when `SignifyToggleFold!` is called.